docs/config.md

```diff
@@ -237,17 +237,18 @@ Important environment variables include:
+|-------------------------------------------------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------|
+| `OTEL_DOTNET_AUTO_TRACES_ENABLED`                     | Enables traces.                                                                                                                                                                                                                                                                                                                                                                                    | `true`        |
+| `OTEL_DOTNET_AUTO_OPENTRACING_ENABLED`                | Enables OpenTracing tracer.                                                                                                                                                                                                                                                                                                                                                                        | `false`       |
+| `OTEL_DOTNET_AUTO_LOGS_ENABLED`                       | Enables logs.                                                                                                                                                                                                                                                                                                                                                                                      | `true`        |
+| `OTEL_DOTNET_AUTO_METRICS_ENABLED`                    | Enables metrics.                                                                                                                                                                                                                                                                                                                                                                                   | `true`        |
+| `OTEL_DOTNET_AUTO_NETFX_ASSEMBLY_REDIRECTION_ENABLED` | Enables automatic redirection of the assemblies used by the automatic instrumentation on the .NET Framework.                                                                                                                                                                                                                                                                                       | `true`        |
```

Comment:
This is correct, although the var name is sooo long. :) Could it be something like:

`OTEL_DOTNET_AUTO_NETFX_ASSEMBLY_REDIRECT` ?

My only question is: Why and when would a user need to disable this? We should add this somewhere.

Reply:
The pattern for boolean env vars is to have the `_ENABLED` suffix. That said I do agree that this is really long... 🤔 perhaps `OTEL_DOTNET_AUTO_NETFX_REDIRECT_ENABLED`? Still on the long side but a bit shorter...

> Why and when would a user need to disable this?

There are some cases that this feature can introduce some runtime errors due to incompatible versions of assemblies. The idea to disable it is mostly to help diagnose problems caused by the feature itself. In such cases, it is possible to suggest workarounds that require the feature to be disabled.

Reply:
If it's not immediately evident to mid-level .NET devs, I'd add it here on in the Troubleshooting docs. Or both.

Reply:
The troubleshooting docs are in need a more careful review. Opened #1863